variable_set.go

```diff
@@ -235,6 +242,72 @@ func (s *variableSets) Delete(ctx context.Context, variableSetID string) error {
 	return s.client.do(ctx, req, nil)
 }
 
+// VariableSetApplyToWorkspacesOptions represents the options for applying variable sets to workspaces.
+type VariableSetApplyToWorkspacesOptions struct {
```

Comment:
We've established a convention for v1.0 that lays out the structure for any given file: 1) Interface/Struct definitions, 2) Receiver methods and 3) validate/helper methods.

Reply:
Just to be clear, it should be _all_ structs at the top of the file, followed by receiver methods and so on...

Reply:
Cheers, happy to move these. To be clear: would you like me to move just these newly-added structs, or all previously-existing ones too? It looks like workspace.tf follows the convention you're describing as a matter of prior art.

Reply:
Yes if you would not mind reorganizing the file that would be great! 🚀

Reply:
Addressed, but this may make the diff somewhat more difficult to read. I've split the re-arrangement into its own commit, here: https://github.com/hashicorp/go-tfe/pull/375/commits/ba9fb5dafb989f5242dfb8bcb7d7fb10e7830be1